app/src/main/java/nu/yona/app/utils/AppUtils.java

```diff
@@ -548,6 +553,7 @@ public void onError(Object errorMessage) {
                 buf.close();
                 return bytes;
             } catch (FileNotFoundException e) {
+                AppUtils.reportException(AppUtils.class.getSimpleName(), e, Thread.currentThread(), null);
                 printStackTrace(e);
             } catch (IOException e) {
                 printStackTrace(e);
```

Comment:
This one should report the exception rather than print its stack trace.

Reply:
Updated.